Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "_lowtime_rs"
+version = "0.1.0"
```

Comment:
Is it possible to not even have a version for this? If not, let's synchronize the version of this one with Python.

Reply:
Seems to be removable. Not completely sure on the implications of not having a version, though.